packages/fiddlerclassic.vm/fiddlerclassic.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>fiddlerclassic.vm</id>
-    <version>5.0.20211.20221209</version>
+    <version>5.0.20211.20240410</version>
```

Comment:
This depedency is wrongly updated, likely related to https://github.com/mandiant/VM-Packages/issues/972